components/kyma-environment-broker/internal/process/provisioning/create_runtime.go

```diff
@@ -60,6 +60,7 @@ func (s *CreateRuntimeStep) Run(operation internal.ProvisioningOperation, log lo
 		log.Errorf("Unable to create provisioning input: %s", err.Error())
 		return s.operationManager.OperationFailed(operation, "invalid operation data - cannot create provisioning input", log)
 	}
+	requestInput.ClusterConfig.Administrators = []*string{&operation.ProvisioningParameters.ErsContext.UserID}
```

Comment:
request send to the Provisioner is built in [input](https://github.com/kyma-project/control-plane/blob/main/components/kyma-environment-broker/internal/process/input/input.go) object by the `CreateProvisionRuntimeInput()` method (see create runtime [step](https://github.com/kyma-project/control-plane/blob/main/components/kyma-environment-broker/internal/process/provisioning/create_runtime.go#L139)).
(`Input` also has a method for building requests for the upgrade process.)
IMO best place for add this parameter will be [`applyProvisioningParametersForProvisionRuntime` method](https://github.com/kyma-project/control-plane/blob/main/components/kyma-environment-broker/internal/process/input/input.go#L219) e.g.:
```go
(...)
updateString(r.provisionRuntimeInput.ClusterConfig.GardenerConfig.Purpose, params.Purpose)
if params.LicenceType != nil {
    r.provisionRuntimeInput.ClusterConfig.GardenerConfig.LicenceType = params.LicenceType
}
r.provisionRuntimeInput.Administrators = []string{r.provisioningParameters.ErsContext.UserID}

r.hyperscalerInputProvider.ApplyParameters(r.provisionRuntimeInput.ClusterConfig, r.provisioningParameters)

return nil
```

Reply:
Done